csv-validation-tests/src/main/shell/GenerateTests.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'open-uri'
```

Comment:
Have you just taken this file from the CsvLint.rb project? If so I suspect we might need to provide some attribution. It would probably be a good idea either way. At the minimum, this file needs to be licensed under the exact same license as where it was taken from.

Reply:
I have added a MIT license to this project, see PR #22.  Original csvlint project is licensed under MIT. Regarding attribution, how exactly? Shall I just use comments in the beginning of this script?

Reply:
I think attribution would just be putting a comment at the beginning saying where it came from originally, ideally with a URL.

We'll wait and see what the result of #22 is to see whether we need to license these specific files copied from the CSVLint project under MIT.